src/components/loading/loading.js

```diff
@@ -1,78 +1,74 @@
 
-                document.body.appendChild(elem);
+        document.body.appendChild(elem);
 
-                layer1 = elem.querySelector('.mdl-spinner__layer-1');
-                layer2 = elem.querySelector('.mdl-spinner__layer-2');
-                layer3 = elem.querySelector('.mdl-spinner__layer-3');
-                layer4 = elem.querySelector('.mdl-spinner__layer-4');
+        layer1 = elem.querySelector('.mdl-spinner__layer-1');
+        layer2 = elem.querySelector('.mdl-spinner__layer-2');
+        layer3 = elem.querySelector('.mdl-spinner__layer-3');
+        layer4 = elem.querySelector('.mdl-spinner__layer-4');
 
-                circleLefts = elem.querySelectorAll('.mdl-spinner__circleLeft');
-                circleRights = elem.querySelectorAll('.mdl-spinner__circleRight');
-            }
+        circleLefts = elem.querySelectorAll('.mdl-spinner__circleLeft');
+        circleRights = elem.querySelectorAll('.mdl-spinner__circleRight');
+    }
 
-            elem.classList.add('mdlSpinnerActive');
+    elem.classList.add('mdlSpinnerActive');
 
-            layer1.classList.add('mdl-spinner__layer-1-active');
-            layer2.classList.add('mdl-spinner__layer-2-active');
-            layer3.classList.add('mdl-spinner__layer-3-active');
-            layer4.classList.add('mdl-spinner__layer-4-active');
+    layer1.classList.add('mdl-spinner__layer-1-active');
+    layer2.classList.add('mdl-spinner__layer-2-active');
+    layer3.classList.add('mdl-spinner__layer-3-active');
+    layer4.classList.add('mdl-spinner__layer-4-active');
 
-            var i;
-            var length;
+    for (let i = 0, length = circleLefts.length; i < length; i++) {
+        circleLefts[i].classList.add('mdl-spinner__circleLeft-active');
+    }
 
-            for (i = 0, length = circleLefts.length; i < length; i++) {
-                circleLefts[i].classList.add('mdl-spinner__circleLeft-active');
-            }
+    for (let i = 0, length = circleRights.length; i < length; i++) {
+        circleRights[i].classList.add('mdl-spinner__circleRight-active');
+    }
+}
 
-            for (i = 0, length = circleRights.length; i < length; i++) {
-                circleRights[i].classList.add('mdl-spinner__circleRight-active');
-            }
-        },
-        hide: function () {
-            var elem = loadingElem;
+function hide() {
```

Comment:
_not tested_
```suggestion
export function hide() {
```

Reply:
If you want, I can add it, but it works with/without it